BL_table/BL_table.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+title: "Make a baseline table"
```

Comment:
General comment: I would name all R chunks
There are some typos and little language problems, but those could be corrected in the end

Reply:
done